conf/DEV.public.conf

```diff
@@ -30,7 +24,19 @@ paypal {
         baseReturnUrl="https://contribute.thegulocal.com"
         paypalWebhookId="some_wehbhook_id"
     }
+
+}
+
+cors {
+    allowedOrigins = [
+        "https://support.thegulocal.com"
+    ]
+}
+
+support-frontend {
+    thank-you-url = "https://support.thegulocal.com/oneoff-contributions/thankyou"
```

Comment:
After an offline discussion with @joelochlann and @davidfurey we preferred the current option over redirecting the user to a page that is sent via an argument.